# add timestamp
Header header

# vehicle id
int32 id

# vehicle state namespace
string nsState

# vehicle insertion or removal
bool insert